lib/LedgerSMB/PGDate.pm

```diff
@@ -294,14 +298,18 @@ sub to_output {
     $fmt = $formats->{uc($fmt)}->[0] if defined $formats->{uc($fmt)};
 
     $fmt .= ' %T' if $self->is_time();
+    $fmt .= '%z' if $self->is_tz();
     $fmt =~ s/^\s+//;
-
+#use Data::Printer;
+#warn p($self);
+#warn $fmt;
```

Comment:
These lines can be (and should be) removed as they clutter the view on the code-flow. (Anybody can re-add them, if they want/need).